plugins/woocommerce-blocks/assets/js/blocks/product-gallery/inner-blocks/product-gallery-thumbnails/edit.tsx

```diff
@@ -2,66 +2,54 @@
  * External dependencies
  */
 import { InspectorControls, useBlockProps } from '@wordpress/block-editor';
-import { Disabled, PanelBody } from '@wordpress/components';
+import { PanelBody } from '@wordpress/components';
 import { WC_BLOCKS_IMAGE_URL } from '@woocommerce/block-settings';
+import type { BlockEditProps } from '@wordpress/blocks';
 import clsx from 'clsx';
 
 /**
  * Internal dependencies
  */
-import './editor.scss';
 import { ProductGalleryThumbnailsBlockSettings } from './block-settings';
-import type { ProductGalleryContext } from '../../types';
+import type { ProductGalleryThumbnailsBlockAttributes } from './types';
 
-interface EditProps {
-	context: ProductGalleryContext;
-}
-
-export const Edit = ( { context }: EditProps ) => {
+export const Edit = ( {
+	attributes,
+	setAttributes,
+}: BlockEditProps< ProductGalleryThumbnailsBlockAttributes > ) => {
 	const blockProps = useBlockProps( {
 		className: clsx(
```

Comment:
We can improve this and remove the `clsx` dependency.
```suggestion
		className: `wc-block-product-gallery-thumbnails wc-block-product-gallery-thumbnails--number-of-thumbnails-${ attributes.numberOfThumbnails }`,
```